core/src/main/scala/kafka/coordinator/GroupMetadata.scala

```diff
@@ -128,6 +129,14 @@ case class GroupSummary(state: String,
                         members: List[MemberSummary])
 
 /**
+ * Case class used to represent group metadata (including the group coordinator) for the DescribeGroup API
+ */
+case class ConsumerGroupSummary(state: String,
```

Comment:
It's a little weird to locate this in `kafka.coordinator` since the coordinator is technically agnostic to group internals. The best alternative I can think of is maybe to put it in tools with `AdminClient`. What do you think?